week3/final/robot-text/robot.js

```diff
@@ -36,8 +40,9 @@
       }
       console.log(line);
     }
+    if (appleEaten) { console.log('YUM!');}
```

Comment:
It is better to always format an `if` statement using multiple line, like this:

```js
if (appleEaten) { 
   console.log('YUM!');
}
```

This makes it easier to add more lines to the `if` block should that be necessary in the future.